flink-ml-tests/src/test/java/org/apache/flink/test/iteration/UnboundedStreamIterationITCase.java

```diff
@@ -152,6 +160,39 @@ public void testVariableAndConstantBoundedIteration() throws Exception {
         assertEquals(OutputRecord.Event.TERMINATED, result.get().take().getEvent());
     }
 
+    @Test
+    public void testUnwrapOperatorConfig() throws Exception {
```

Comment:
Can we name this test based on the key property of the API used by this test?

For example, if the key property here is the use of SideOutput, maybe name it as `testBoundedIterationWithSideoutput`.